draft-iab-privacy-partitioning.md

```diff
@@ -446,29 +446,55 @@ about individual client data.
+The most impactful types of information to partition are (a) user-identifying information,
+such as user identity or identities (including account names or IP addresses) that can be
+linked and (b) non-user-identifying information, or user data (including the
+content a user is accessing), which can be often sensitive when combined with user identity.
+
+In this section, we discuss considerations for partitioning these types of information.
+
+## Partitioning Considerations
+
+User data can itself be user-identifying, in which case it should be treated as an identifier.
 For example, Oblivious DoH and Oblivious HTTP partition the client IP address and client request data into
 separate contexts, thereby ensuring that no entity beyond the client can observe both. Collusion across contexts
 could reverse this partitioning, but can also promote non-user-identifying information to user-identifying.
 For example, in CONNECT proxy systems that use QUIC, the QUIC connection ID is inherently non-user-identifying
 since it is generated randomly ({{?QUIC=RFC9000, Section 5.1}}). However, if combined with another context that has user-identifying
 information such as the client IP address, the QUIC connection ID can become user-identifying information.
 
-This partitioning process can be applied incorrectly or incompletely. Contexts may contain
+Some information is innate to client user-agents, including includes details of implementation of
+protocols in hardware and software, and network location. This information can be used to construct
+user-identifying information, which is a process sometimes referred to as fingerprinting.
+Depending on the application and system constraints, users may not be able to prevent fingerprinting
+in privacy contexts. As a result, fingerprinting information, when combined with non-user-identifying
+user data, could promote user data to user-identifying information.
+
+## Incorrect or Incomplete Partitioning
+
+Privacy partitioning can be applied incorrectly or incompletely. Contexts may contain
 more user-identifying information than desired, or some information in a context may be more user-identifying
-than intended. Moreover, splitting user-identifying information over multiple contexts has to be done with care, as creating more contexts can increase the number of entities that need to be trusted to not collude.
+than intended. Moreover, splitting user-identifying information over multiple contexts has to be done
+with care, as creating more contexts can increase the number of entities that need to be trusted to not collude.
 Nevertheless, partitions can help improve the client's privacy posture when applied carefully.
 
+## Incorrect or Incomplete Partitioning
```

Comment:
```suggestion
```
Do you have this heading twice or am I not able to read the diff correctly?

Reply:
I also see a duplicate heading. @chris-wood What was the intention?